### PR TITLE
add matcher for `have_secure_token`

### PR DESCRIPTION
**Pull request by kuldeepaggarwal**
Please help me to add specs. Not sure how to add that.

### CODE REVIEW

File: lib/shoulda/matchers/active_record/have_secure_token.rb [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+      #
+      #     # Minitest (Shoulda)
+      #     class UserTest < ActiveSupport::TestCase
+      #       should have_secure_token
+      #       should have_secure_token(:public_id)
+      #     end
+      #
+      # @return [HaveSecurePasswordMatcher]
+      #
+      def have_secure_token(attr = :token)
```
Comment (kuldeepaggarwal) [inline lib/shoulda/matchers/active_record/have_secure_token.rb]: Why so?

**Comment by ianks** [comment]
This would be nice.

**Comment by Techbrunch** [comment]
This should probably be closed in favor of #1025

**Comment by mcmire** [comment]
Agreed, thank you :)

**Pull request closed without merging**